src/Tribe/Aggregator.php

```diff
@@ -58,8 +58,7 @@ class Tribe__Events__Aggregator {
 	 */
 	public static function instance() {
 		if ( ! isset( self::$instance ) ) {
```

Comment:
I'm not sure all on a single line is easier to read. :wink: 

I still think it makes more sense to declare the var with a value, especially if we're checking for the value. Otherwise I would probably use `empty()` and not `isset()` as it's a bit more forgiving.